### ISSUE TITLE
Use petab validator and restrict access to main branch

### ISSUE THREAD

**Issue by yannikschaelte** [LeonardSchmiester/Benchmark-Models#62]
I'd suggest: At some point, https://github.com/icb-dcm/petab should be run automatically as a test and used to check every new model before adding it to the official list of models, or essentially every pull request, also restricting access to the main branch (but only once all are fine, not now yet ;) )

**Comment by dweindl** [comment]
I'd add that right away so that on every commit to whatever branch we see which models are fine and which are not. Volunteers?

**Comment by dweindl** [comment]
Validator now running on travis, but will succeed despite validation passing. Change when most models have been fixed.
